03-interactive-site/02-ux-design/09-ux-challenges/README.md

```diff
@@ -39,10 +39,9 @@ Um exemplo de um dashboard em outro contexto é o seguinte:
 
 ![](https://cdn.colorlib.com/wp/wp-content/uploads/sites/2/free-bootstrap-admin-dashboard-templates.jpg)
 
-**Lembre-se** que este esboço é parte do [produto final](01-final-product) desta unidade.
+**Lembre-se** que este esboço é parte do [produto final](../03-data-dashboard/01-final-product) desta unidade.
```

Comment:
Here, I think the LMS just recognize folder names, so I think we'll need to fix this, but I will accept the PR and then let's see if we need another PR to solve this.

Reply:
I tested it localhost and it didn't work, that is the reason I have changed it =) I have just tested it in staging and it is work 🎉